spec/server_spec.lua

```diff
@@ -13,14 +13,24 @@ describe("http.server module", function()
 			error(err, 2)
 		end
 	end
-	local function simple_test(tls, version)
+	local function simple_test(tls, version, path)
 		local cq = cqueues.new()
-		local s = server.listen {
-			host = "localhost";
-			port = 0;
-		}
+		local options = {}
+		if path then
+			options.path = path
+		else
+			options.host = "localhost"
+			options.port = 0
+		end
+		options.version = version
+		options.tls = tls
```

Comment:
Confirmed tonight that it does; instead it assumes (at least on my system) that it needs to generate a certificate (which breaks atm because certificates are pending).